packages/store/src/keys/copyMap.js

```diff
@@ -56,10 +56,26 @@ export const checkCopyMap = (m, check = x => x) => {
 };
 harden(checkCopyMap);
 
+/**
+ * @callback IsCopyMap
+ * @param {Passable} s
```

Comment:
Minor nit: I think both here and below, the param `s` should be `m` instead in the JSDoc.

Reply:
Good catch